Add an indicator for rename operations.

compholio opened the pull request:
Without this a staged rename operation does not show up in any of the status indicators.

compholio committed:
Add an indicator for rename operations.

cy4n commented:
i like the change, but the icon does not work for me
![screen shot 2017-10-24 at 09 27 24](https://user-images.githubusercontent.com/1135762/31929828-a5baceb4-b89d-11e7-8c39-c7a089a1e236.png)

cy4n commented:
what iconset or font do you use? @compholio

compholio commented:
The custom one based on Awesome Terminal Fonts...
![tmp](https://user-images.githubusercontent.com/1738066/31942410-69588b52-b882-11e7-902a-ec0fb8e3e9f1.png)

I'm open to suggestion for the icon, the arrow was just the only thing I was seeing that sort of made sense.

compholio commented:
Do you have U+E695?  It looks virtually identical.

cy4n commented:
http://www.fileformat.info/info/unicode/char/e695/index.htm <http://www.fileformat.info/info/unicode/char/e695/index.htm>
„is not a valid unicode char“

leave it this way, ill search for a one thats my shell can displey.

(after doing a quick google search i am actually tempted to replace every icon with utf-8 emoji from this list: :-)
https://unicode.org/emoji/charts/full-emoji-list.html <https://unicode.org/emoji/charts/full-emoji-list.html>

compholio commented:
Sorry it's taken me so long to get back to this... I pushed a fix for the symbol not showing/hiding correctly:
https://github.com/compholio/oh-my-git/commit/ad4cf403ccb0d5e594899a8686a004d710d2471b

I'm not sure what you mean about move/rename automatically staging though, I don't see that:
![rename](https://user-images.githubusercontent.com/1738066/33782639-b8a4f22e-dc16-11e7-8e9a-95925d961f38.png)

I normally use the "condensed" mode that I added a while back though, which is why I added:
```
if [[ ${omg_condensed} == true && ${has_renames} == true ]]; then
    has_modifications_cached=true
fi
```

cy4n commented:
hi erich,

i misunderstood something, doesnt matter any more though;)
your change works for me now, i just merged it into my master branch.
i did change the arrow to a unicode arror in another commit , to avoid having to add another font pack
cheers!

> On 8. Dec 2017, at 20:55, Erich E. Hoover <notifications@github.com> wrote:
> 
> Sorry it's taken me so long to get back to this... I pushed a fix for the symbol not showing/hiding correctly:
> compholio/oh-my-git@ad4cf40 <https://github.com/compholio/oh-my-git/commit/ad4cf403ccb0d5e594899a8686a004d710d2471b>
> I'm not sure what you mean about move/rename automatically staging though, I don't see that:
>  <https://user-images.githubusercontent.com/1738066/33782639-b8a4f22e-dc16-11e7-8e9a-95925d961f38.png>
> I normally use the "condensed" mode that I added a while back though, which is why I added:
> 
> if [[ ${omg_condensed} == true && ${has_renames} == true ]]; then
>     has_modifications_cached=true
> fi
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub <https://github.com/cy4n/oh-my-git/pull/8#issuecomment-350356981>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ABFUkpPhvWisPVg9OQhD86jTx2JDu8zDks5s-ZQagaJpZM4QB-4a>.
>

compholio commented:
Awesome, I have no idea why my fonts are so weird :/
